compiler/rustc_target/src/spec/targets/x86_64_unknown_fuchsia.rs

```diff
@@ -4,6 +4,9 @@ pub(crate) fn target() -> Target {
     let mut base = base::fuchsia::opts();
     base.cpu = "x86-64".into();
     base.plt_by_default = false;
+    // See https://fuchsia.dev/fuchsia-src/contribute/governance/rfcs/0073_x86_64_platform_requirement,
+    // which corresponds to x86-64-v2.
+    base.features = "+cx16,+sahf,+popcnt,+sse3,+sse4.1,+sse4.2,+ssse3".into();
     base.max_atomic_width = Some(64);
```

Comment:
```suggestion
    base.max_atomic_width = Some(128);
```

cx16 is available so we can use 128-bit atomics.

Reply:
Thanks! Fixed.